recipes/reaper/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "REAPER" %}
+{% set gitversion = "e22f2dbdad3af8b725d3fc25e6c96b23ae9aff70" %}
+{% set version = "0.0.2018.01.13" %}
```

Comment:
Hey, sorry for the long delay. Can we number by commit date instead? So `0.0.2017.12.03`.